test/mix/tasks/openapi.spec.json_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule Mix.Tasks.Openapi.Spec.JsonTest do
+  use ExUnit.Case
+
+  test "generates openapi.json" do
+    actual_schema_path = "tmp/openapi.json"
+
+    Mix.Tasks.Openapi.Spec.Json.run(~w(
+      --pretty=true
+      --spec OpenApiSpexTest.Tasks.SpecModule
+      --vendor-extensions=false
+      #{actual_schema_path}
+    ))
+
+    expected_schema_path = "test/support/tasks/openapi.json"
+
+    assert_received {:mix_shell, :info, ["* creating tmp"]}
```

Comment:
@lazebny I pushed a commit to silence as well as test the console output. Otherwise it prints the following when running the tests.

![Screenshot 2022-07-03 at 19 35 24](https://user-images.githubusercontent.com/645514/177052851-83d186ed-1f46-41b2-a6d6-54e67cdcc5dd.png)

Reply:
Makes sense!